button.py

```diff
@@ -0,0 +1,47 @@
+from constants import *
+
+class Button(pygame.sprite.Sprite):
+    def __init__(self, button_text, center_x, center_y):
+        super().__init__()
+        self.value = button_text
+        self.clicked = False
+        self.button_color = 'white'
+        self.shadow_color = CLAIRVOYANT
+
+        #create transparent button surface
+        self.image = pygame.Surface((250, 60), pygame.SRCALPHA)
+        self.rect = self.image.get_rect(centerx=center_x, centery=center_y)
+
+    def update(self):
+        #"clear" button image of previously draw elements
+        self.image.fill(pygame.Color(0,0,0,0))
```

Comment:
Yes, it's a transparent black so that the background can still be seen.  I can't create a surface with rounded corners, so I had to draw a rectangle with rounded corners on top of the button surface.  This transparency allows us to only see the rounded corners of the drawn rectangle and not the sharp corner of the surface.

Reply:
Ah gotcha, that makes sense, thanks!